engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content-hosts-helper.service.js

```diff
@@ -69,20 +56,24 @@ angular.module('Bastion.content-hosts').service('ContentHostsHelper',
             var colors = {
                     'valid': 'green',
                     'partial': 'yellow',
-                    'invalid': 'red'
+                    'invalid': 'red',
+                    0: 'green',
+                    1: 'yellow',
+                    2: 'red',
+                    3: 'red'
                 };
 
             return colors[status] ? colors[status] : 'red';
         };
 
-        this.getProvisioningStatusColor = function (status) {
-            var color;
-            if (angular.isDefined(status)) {
-                if (angular.isUndefined(color = hostStatusColorMap[status])) {
-                    throw "Unknown status = " + status;
-                }
-            }
-            return color;
+        this.getGlobalStatusColor = function (status) {
+            var colors = {
+                    0: 'green',
+                    1: 'yellow',
+                    2: 'red'
+                };
```

Comment:
they aren't entirely, getStatusColor is for subscriptions only, it was never really named properly.  I will rename it